sdv/metadata/table.py

```diff
@@ -698,6 +699,9 @@ def reverse_transform(self, data):
             else:
                 field_data = reversed_data[name]
 
+            if field_metadata['transformer'] == 'integer':
+                field_data = field_data.round()
```

Comment:
can we add a unit test for this line?

Reply:
Sure. I didn't add any because there were no reverse_transform tests in the first place, presumably because it gets tested by the integration ones.